crud/common/utils.lua

```diff
@@ -524,4 +581,14 @@ function utils.flatten_obj_reload(space_name, obj)
     return schema.wrap_func_reload(flatten_obj, space_name, obj)
 end
 
+function utils.get_index_fieldno_map(index_obj)
+    local t = {}
```

Comment:
Name ``t`` is not very clear.

Reply:
Which one do you prefer?
I think one letter variable is acceptable for a function body with 4 lines.